### PR TITLE
add default output of telegraf

### PR DESCRIPTION
**Pull request by yanganto**

### CODE REVIEW

**Comment by yanganto** [comment]
Just wondering why the test did not fail without this PR
https://github.com/kuutamolabs/near-staking-knd/blob/b830eb16f54b6cf8f9ef2084d15cd900bf9621cf/nix/modules/tests/monitoring.nix#L49-L50

File: nix/modules/telegraf.nix [hunk 1]
```diff
@@ -29,8 +29,12 @@
...
           ];
         };
-        outputs = lib.optionalAttrs config.kuutamo.telegraf.hasMonitoring {
-          http = {
+        outputs = {
+          prometheus_client = {
+            listen = ":9273";
+            metric_version = 2;
+          };
+          http = lib.mkIf config.kuutamo.telegraf.hasMonitoring {
```
Comment (Mic92) [inline nix/modules/telegraf.nix:37]: simplified this a bit.

File: nix/modules/telegraf.nix [hunk 2]
```diff
@@ -29,8 +29,12 @@
             "http://localhost:2233/metrics"
           ];
         };
-        outputs = lib.optionalAttrs config.kuutamo.telegraf.hasMonitoring {
-          http = {
+        outputs = {
+          prometheus_client = {
```
Comment (Mic92) [inline nix/modules/telegraf.nix:33]: I think this is actually already provided by srvos: https://github.com/numtide/srvos/blob/9e172d6cd8afffef25e46208df88a2946f1f8f0d/nixos/mixins/telegraf.nix#L129

Comment (Mic92) [reply nix/modules/telegraf.nix]: Even after dropping it still seems to be included:

```
[agent]
interval = "60s"

[inputs]
[[inputs.exec]]
commands = ["/nix/store/b4h077s4afivvsl32a366f3piqvzmj0f-ipv6-dad-check"]
data_format = "influx"

[[inputs.file]]
data_format = "influx"
file_tag = "name"
files = ["/var/log/telegraf/*"]

[[inputs.file]]
data_format = "value"
files = ["/sys/fs/ext4/*/errors_count"]
name_override = "ext4_errors"

[inputs.disk.tagdrop]
device = ["rpc_pipefs", "lxcfs", "nsfs", "borgfs"]
fstype = ["tmpfs", "ramfs", "devtmpfs", "devfs", "iso9660", "overlay", "aufs", "squashfs"]

[inputs.diskio]

[inputs.kernel_vmstat]

[inputs.mdstat]

[inputs.mem]

[inputs.prometheus]
metric_version = 2
urls = ["http://localhost:3030/metrics", "http://localhost:2233/metrics"]

[inputs.smart]
path = "/nix/store/sgywm45hiwmxjqj1s4cwr806hhaf1p7s-smartctl"

[inputs.swap]

[inputs.system]

[inputs.systemd_units]

[outputs.prometheus_client]
listen = ":9273"
metric_version = 2
```

**Review by Mic92 — APPROVED** [review]

**Pull request merged**